### DISCUSSION TITLE
Vuetify 3.1.2 Grid: fit column height

### DISCUSSION THREAD

**Discussion by AniaKru95** [vuetifyjs/vuetify#16688] [Q&A] [answered]
Hi!
Is there a way to fit column height to sibling height?

There's an example. I would like to fit height Card1 to Card2.
https://codepen.io/AniaKru95/pen/yLxyWgz?editors=101

I don't know a fixed height. It could be dynamic, sometimes Card2 should fit height to Card1.

**Comment by yuwu9145** [comment, marked as answer]
Use [Vuetify utility class `align-stretch` ](https://vuetifyjs.com/en/styles/flex/#flex-align)

https://codepen.io/yuchaosydney/pen/ExeEmmJ?editors=1010

**Reply by AniaKru95** [reply]
It works, thank you :)
